interfaces/apparmor/template.go

```diff
@@ -489,6 +489,9 @@ var classicJailmodeSnippet = `
   # Read only access to the core snap to load libc from.
   # This is related to LP: #1666897
   @{INSTALL_DIR}/core/*/{,usr/}lib/@{multiarch}/{,**/}lib*.so* m,
+
+  # Read only access to the core snap-exec (needed on bionic)
+  @{INSTALL_DIR}/core/*/usr/lib/snapd/snap-exec m,
```

Comment:
FYI, this isn't read-only access, it is mmap with PROT_EXEC. template.go already has this comment, I suggest we use the same here:
```
# For snappy reexec on 4.8+ kernels
```